examples/example-app-router-playground/src/i18n/request.tsx

```diff
@@ -1,9 +1,31 @@
 import {headers} from 'next/headers';
 import {notFound} from 'next/navigation';
+import type {Formats} from 'next-intl';
```

Comment:
Is there a benefit here from your perspective for using `import type`?

For simplicity, I think I'd leave the `type` part out—that's at least what I usually do.